src/IECoreMaya/FromMayaArrayDataConverter.cpp

```diff
@@ -38,11 +38,33 @@
 
 #include "IECore/VectorTypedData.h"
 
+#include "IECore/GeometricTypedData.h"
+
 using namespace IECore;
 
 namespace IECoreMaya
 {
 
+namespace Detail
```

Comment:
Let's let @andrewkaufman decide, but at a `\todo` item if we decide not to do it right now.